helm_deploy/hmpps-send-legal-mail-staff-ui/values.yaml

```diff
@@ -58,6 +58,7 @@ generic-service:
     cloudplatform-live-1: "35.178.209.113/32"
     cloudplatform-live-2: "3.8.51.207/32"
     cloudplatform-live-3: "35.177.252.54/32"
+    moj-official-tgw-prod: "51.149.250.0/24"
```

Comment:
Added this one as Simon was unable to access the application from a Windows box.. Copied from vsip-admin-ui app